### PR TITLE
Add Grove archival endpoints to `nodes/mainnet.md`

### PR DESCRIPTION
**Pull request by Olshansk**
## Overview

- Add links to [Celestia on Grove](https://docs.grove.city/celestia-api/intro) and the [supported endpoints](https://docs.grove.city/celestia-api/endpoints/celestia-archival).
- Add an image of how to use Grove's sandbox for testing
- s/`Mainnet Beta`/`Mainnet`

## Summary by CodeRabbit

- **Documentation**
	- Renamed "Mainnet Beta" to "Mainnet" across the guide to reflect the transition to a stable production network.
	- Updated the guide to focus on the evolution and stability of Celestia's Mainnet.
	- Added information on types of nodes and details on Data Availability (DA) RPC endpoints and Archival DA RPC endpoints.

### CODE REVIEW

File: nodes/mainnet.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
 <!-- markdownlint-disable MD033 -->
 
-# Mainnet Beta
```
Comment (jcstein) [inline nodes/mainnet.md:3]: please revert all of these instances back to `Mainnet Beta`

**Review by jcstein — CHANGES_REQUESTED** [review]

File: nodes/mainnet.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+# Mainnet
```
Comment (jcstein) [inline nodes/mainnet.md:3]:
```suggestion
# Mainnet Beta
```

File: nodes/mainnet.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 
 <script setup>
 import MainnetVersionTags from '../.vitepress/components/MainnetVersionTags.vue'
 import MainnetBetaDetails from '../.vitepress/components/MainnetBetaDetails.vue'
 </script>
 
-![Mainnet Beta](/img/Mainnet-Beta.png)
+![Mainnet](/img/Mainnet-Beta.png)
```
Comment (jcstein) [inline nodes/mainnet.md:10]:
```suggestion
![Mainnet Beta](/img/Mainnet-Beta.png)
```

File: nodes/mainnet.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
 
-Welcome to the guide for Celestia’s Mainnet Beta, the production
+Welcome to the guide for Celestia’s Mainnet, the production
```
Comment (jcstein) [inline nodes/mainnet.md:12]:
```suggestion
Welcome to the guide for Celestia’s Mainnet Beta, the production
```

File: nodes/mainnet.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
 network that marks the pinnacle of Celestia’s evolution since its
 inception in 2019. This network is where all components of the
 Celestia ecosystem come to life in a real-world environment.
 
-Mainnet Beta is the culmination of rigorous community testing,
+Mainnet is the culmination of rigorous community testing,
```
Comment (jcstein) [inline nodes/mainnet.md:17]:
```suggestion
Mainnet Beta is the culmination of rigorous community testing,
```

File: nodes/mainnet.md [hunk 6]
```diff
@@ -1,26 +1,26 @@
...
 upgrades, and feedback. It serves as the platform for deploying
 mainnet rollups and applications.
 
 ## Network stability and upgrades
 
-Mainnet Beta is a stable network, but will still receive updates and
+Mainnet is a stable network, but will still receive updates and
```
Comment (jcstein) [inline nodes/mainnet.md:23]:
```suggestion
Mainnet Beta is a stable network, but will still receive updates and
```

File: nodes/mainnet.md [hunk 7]
```diff
@@ -84,15 +84,15 @@ block to vary.
 
 ## Integrations
 
-This guide contains the relevant sections for how to connect to Mainnet Beta,
+This guide contains the relevant sections for how to connect to Mainnet,
```
Comment (jcstein) [inline nodes/mainnet.md:87]:
```suggestion
This guide contains the relevant sections for how to connect to Mainnet Beta,
```

File: nodes/mainnet.md [hunk 8]
```diff
@@ -84,15 +84,15 @@ block to vary.
...
 depending on the type of node you are running. Your best approach to
 participating is to first determine which node you would like to run. Each
 node’s guide will link to the relevant network in order to show you how
 to connect to them. Learn about the different endpoint types
 [in the Cosmos SDK documentation](https://docs.cosmos.network/v0.50/learn/advanced/grpc_rest).
 
 Here is a list of options of the types of nodes you can run in order
-to participate in Mainnet Beta:
+to participate in Mainnet:
```
Comment (jcstein) [inline nodes/mainnet.md:95]:
```suggestion
to participate in Mainnet Beta:
```

File: nodes/mainnet.md [hunk 9]
```diff
@@ -205,11 +205,10 @@ celestia <da_type> start --core.ip <url> –core.rpc.port <port> \
 :::tip Bridge nodes
 Not all RPC endpoints guarantee the full block history.
 Find [an archive endpoint on the community dashboard](https://celestia-tools.brightlystake.com/)
-or run your own full consensus node with no pruning for
-your bridge node.
+or run your own full consensus node with no pruning for your bridge node.
 :::
 
-RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with:
+RPCs for DA nodes to initialise or start your celestia-node to Mainnet with:
```
Comment (jcstein) [inline nodes/mainnet.md:211]:
```suggestion
RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with:
```

File: nodes/mainnet.md [hunk 10]
```diff
@@ -234,9 +233,19 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
 
 You can [find the status of these endpoints](https://celestia-tools.brightlystake.com/).
 
+#### Archival DA RPC endpoints
+
```
Comment (jcstein) [inline nodes/mainnet.md:237]: - [ ] add context on what these type of endpoints are before jumping straight to Grove specifics

File: nodes/mainnet.md [hunk 11]
```diff
@@ -234,9 +233,19 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
...
+You can provision your own Celestia Archival endpoint on [Grove](https://grove.city).
```
Comment (jcstein) [inline nodes/mainnet.md:238]:
```suggestion
You can [provision your own Celestia Archival endpoint on Grove](https://grove.city).
```

File: nodes/mainnet.md [hunk 12]
```diff
@@ -234,9 +233,19 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
...
+Learn more about Celestia on Grove [here](https://docs.grove.city/celestia-api/intro),
```
Comment (jcstein) [inline nodes/mainnet.md:239]:
```suggestion
[Learn more about Celestia on Grove](https://docs.grove.city/celestia-api/intro),
```

File: nodes/mainnet.md [hunk 13]
```diff
@@ -234,9 +233,19 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
...
+or find the fully supported spec [here](https://docs.grove.city/celestia-api/endpoints/celestia-archival).
```
Comment (jcstein) [inline nodes/mainnet.md:240]:
```suggestion
or find [the fully supported spec](https://docs.grove.city/celestia-api/endpoints/celestia-archival).
```

File: nodes/mainnet.md [hunk 14]
```diff
@@ -234,9 +233,19 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
...
+
+There is a sandbox you can leverage fo testing straight in your browser:
+
+<img src="https://github.com/celestiaorg/docs/assets/1892194/f6933aba-1fb2-4e02-a3c8-af02d7db3f21" width="600" height="600" alt="sandbox">
```
Comment (jcstein) [inline nodes/mainnet.md:244]:
```suggestion
![grove-sandbox(/path-to-image-for-this.png)
```

File: nodes/mainnet.md [hunk 15]
```diff
@@ -234,9 +233,19 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
...
+
 ## Explorers
 
-There are multiple explorers you can use for Mainnet Beta:
+There are multiple explorers you can use for Mainnet:
```
Comment (jcstein) [inline nodes/mainnet.md:248]:
```suggestion
There are multiple explorers you can use for Mainnet Beta:
```

File: nodes/mainnet.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jcstein) [reply nodes/mainnet.md]: it's used still. ignore this.

File: nodes/mainnet.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jcstein) [reply nodes/mainnet.md]: no. it is still Mainnet Beta

File: nodes/mainnet.md (hunk 1 above)
Comment (jcstein) [reply nodes/mainnet.md]: done in https://github.com/celestiaorg/docs/pull/1525/commits/7be6097074cc56d136ce09a1b3b94a79c9c317c7

File: nodes/mainnet.md (hunk 14 above)
Comment (jcstein) [reply nodes/mainnet.md]: suggest adding the photo to `public/rpcs/grove.png` or something of the sort!

**Comment by jcstein** [comment]
last thing and then this is good to 🚢 

> - [ ] add context on what these type of endpoints are before jumping straight to Grove specifics

File: nodes/mainnet.md (hunk 10 above)
Comment (Olshansk) [reply nodes/mainnet.md:237]: Done.

File: nodes/mainnet.md [hunk 16]
```diff
@@ -234,6 +233,23 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
 
 You can [find the status of these endpoints](https://celestia-tools.brightlystake.com/).
 
+#### Archival DA RPC endpoints
+
+By default, light nodes prune recent data to save on storage space. Archival
+Data Availability (DA) nodes store the entire history of the chain without
```
Comment (jcstein) [inline nodes/mainnet.md:239]:
```suggestion
data availability (DA) nodes store the entire history of the chain without
```

File: nodes/mainnet.md [hunk 17]
```diff
@@ -184,7 +184,7 @@ to participate in Mainnet Beta:
 - [Bridge node](./bridge-node.md)
 - [Full storage node](./full-storage-node.md)
 
-#### DA RPC endpoints
+#### Data Availability (DA) RPC endpoints
```
Comment (jcstein) [inline nodes/mainnet.md:187]:
```suggestion
#### Data availability (DA) RPC endpoints
```

File: nodes/mainnet.md [hunk 18]
```diff
@@ -234,6 +233,23 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
 
 You can [find the status of these endpoints](https://celestia-tools.brightlystake.com/).
 
+#### Archival DA RPC endpoints
+
+By default, light nodes prune recent data to save on storage space. Archival
+data availability (DA) nodes store the entire history of the chain without
+pruning any data so all data available data is retrievable. You can read more
```
Comment (jcstein) [inline nodes/mainnet.md:240]:
```suggestion
pruning any data so all data available data is retrievable. You can
```

File: nodes/mainnet.md [hunk 19]
```diff
@@ -234,6 +233,23 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
...
+about light vs archival nodes [here](../learn/retrievability.md).
```
Comment (jcstein) [inline nodes/mainnet.md:241]:
```suggestion
[read more about light vs archival nodes](../learn/retrievability.md).
```

File: nodes/mainnet.md [hunk 20]
```diff
@@ -234,6 +233,23 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
...
+
+##### Grove archival endpoints
+
+You can [provision your own Celestia Archival endpoint on Grove](https://grove.city).
+[Learn more about Celestia on Grove](https://docs.grove.city/celestia-api/intro),
+or find [the fully supported spec](https://docs.grove.city/celestia-api/endpoints/celestia-archival).
+
+There is a sandbox you can leverage for testing straight in your browser:
+
+![grove-sandbox](../public/grove/grove-sandbox.png)
```
Comment (jcstein) [inline nodes/mainnet.md:251]:
```suggestion
![grove-sandbox](/grove/grove-sandbox.png)
```

**Pull request merged**